perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -238,6 +238,24 @@ __PACKAGE__->has_many(
   },
 );
 
+__PACKAGE__->might_have(
+  contributed_by => "FixMyStreet::DB::Result::User",
+  sub {
+    my $args = shift;
+    return {
```

Comment:
This could be replaced by something like `$args->{self_alias}.extra->'contributed_by' = $args->{foreign_alias}.id` if column was JSONB. I think.